Framework/DataHandling/src/LoadGSS.cpp

```diff
@@ -447,7 +447,8 @@ API::MatrixWorkspace_sptr LoadGSS::loadGSASFile(const std::string &filename,
                            detectorIDs, totalflightpaths, twothetas);
 
   // Clean up
-  delete prog;
+  if (prog)
+    delete prog;
```

Comment:
Did this really generate a warning? Deleting a `nullptr` is perfectly fine according to the standard.